locale/en/caravan.cfg

```diff
@@ -1,15 +1,19 @@
 [entity-name] 
 caravan=Caravan
+fluidavan=Fluid caravan
 flyavan=Aerial caravan
 nukavan=☢ Caravan ☢
 outpost=Caravan outpost
+fluid-outpost=Caravan fluid outpost
```

Comment:
`outpost-fluid` seems like a more better name given the paradigm is already `outpost`, `outpost-aerial`.